### ISSUE TITLE
Mutation support?

### ISSUE THREAD

**Issue by rpapesch** [ropensci/ghql#20]
I was now using the ghql library for some queries and they are working nicely.  I don't find a related class for mutation operations...

Is there also support for GraphQL mutations?

**Comment by sckott** [comment]
thanks. it looks like mutations change data on the server? If so, I'm not sure mutations would be a good fit here since this pkg is meant only for fetching data, not modifying data. But, maybe we could add this. I haven't played around with creating a graphql instance to test against. if i could then I could test this feature

**Comment by rpapesch** [comment]
yes, I have a scenario where I want to publish statistical summaries to a graphql endpoint. It would be great if this feature could be added in the meantime I'm using a native HTTP call in my R script to solve that.

https://graphql.org/graphql-js/mutations-and-input-types/

**Comment by sckott** [comment]
okay, i'll try soon

**Comment by sckott** [comment]
@friep any interest in looking at this feature request?

**Comment by friep** [comment]
@sckott i'm having a few busy weeks / months ( 😢 ) ahead but in spring (~April) i might have the time.

**Comment by sckott** [comment]
Okay, thanks. If I have time before then i'll try, otherwise go for it when you have time
